acronym.dtx

```diff
@@ -682,8 +682,8 @@ blocks to be tested separately. The latter are commonly indicated as
 %    First we test that we got the right format and name the package.
 %    \begin{macrocode}
 \NeedsTeXFormat{LaTeX2e}[1999/12/01]
-\ProvidesPackage{acronym}[2020/03/13
-                          v1.46
+\ProvidesPackage{acronym}[2020/04/17
```

Comment:
Is the 1.47 release date intentionally in the future?

Reply:
no :) as explained in the PR :) but as it is already released I did not change it back